### PR TITLE
Media not title for tutorial

### LINKED ISSUE #1425: Tutorial steps should not required a title
Re: this error: `{message: "{"steps/0"=>"did not contain a required property of 'title'"}"}`

Each tutorial step should have

- A `media` string, which will be the ID of the `attached_images` link for the step

- A `content` string, which'll just be some Markdown

### PR DESCRIPTION
**Pull request by edpaget**
Closes #1425

Sorry @brian-c

### CODE REVIEW

File: app/schemas/tutorial_update_schema.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class TutorialUpdateSchema < JsonSchema
         additional_properties false
         required "title", "content"
```
Comment (camallen) [inline app/schemas/tutorial_update_schema.rb]: required media?

File: app/schemas/tutorial_create_schema.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@ class TutorialCreateSchema < JsonSchema
         additional_properties false
         required "title", "content"
```
Comment (camallen) [inline app/schemas/tutorial_create_schema.rb]: required media?

Comment (brian-c) [inline app/schemas/tutorial_create_schema.rb]: Neither should be required, actually.

**Comment by edpaget** [comment]
Removed all the validations.

**Pull request merged**